doc/extending.md

```diff
@@ -2,20 +2,20 @@
 
 There are two extension points:
 - ## Support for custom Field Types
-If you want BehatBundle to support your custom Field Type when generating Content items you need to implement `EzSystems\Behat\API\ContentData\FieldTypeData\FieldTypeDataProviderInterface` and tag the service with the `ezplatform.behat.fieldtype_data_provider` tag.
+If you want BehatBundle to support your custom Field Type when generating Content items you need to implement `Ibexa\Behat\API\ContentData\FieldTypeData\FieldTypeDataProviderInterface` and tag the service with the `ibexa.behat.fieldtype_data_provider` tag.
```

Comment:
BehatBundle is now just behat repository, so it should be adjust in all places in this doc.